client/blocks/like-button/README.md

```diff
@@ -28,15 +28,20 @@ render: function() {
 ```js
 import LikeButton from 'blocks/like-button/button';
 
-render: function() {
+function render() {
 	return (
 		<div className="your-stuff">
-			  <LikeButton likeCount={ 5 } showCount={ true } liked={ false } onLikeToggle={ this.handleLikeToggle } />
+			<LikeButton
+				likeCount={ 5 }
+				showCount={ true }
```

Comment:
This is a nit, but `={ true }` can be omitted here.

Reply:
Fixed